vertexy/src/private/prefab/PrefabManager.cpp

```diff
@@ -61,7 +96,76 @@ void PrefabManager::generatePrefabConstraints(const shared_ptr<TTopologyVertexDa
 	// Prefab Constraints
 	for (auto& prefab : m_prefabs)
 	{
-		prefab->generatePrefabConstraints(m_solver, m_grid, tileData);
+		// Ensure we have a real position if we have a prefab
+		m_solver->makeGraphConstraint<ClauseConstraint>(m_grid, ENoGood::NoGood,
+			GraphRelationClause(selfTilePrefab, { prefab->id() }),
+			GraphRelationClause(selfTilePrefabPos, { Prefab::NO_PREFAB_POS })
+			);

```

Comment:
minor: for style consistency, can you align the end-paren with the top of the statement